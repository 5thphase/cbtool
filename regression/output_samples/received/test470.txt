 status: VMC 22A02FDA-3999-5B5A-9EF9-0555D64855AD was successfully registered on SimCloud "TESTCLOUD".
 status: VMC 31D627D0-7733-5110-A077-926AFB38B046 was successfully registered on SimCloud "TESTCLOUD".
 status: VMC BC2B67B7-F8FC-596D-B4B6-89781278A46F was successfully registered on SimCloud "TESTCLOUD".
 status: VMC E737205E-089F-56C4-B594-FBDFC6437474 was successfully registered on SimCloud "TESTCLOUD".
 status: Attribute "collect_from_host" was set to "false". Skipping Host OS performance monitor daemon startup
All VMCs successfully attached to this experiment.
The following ATTACHED VMCs are in this experiment (Cloud TESTCLOUD) :
|name                    |host_count      |pool                    |cloud_hostname                  |cloud_ip         
|simzone_a               |5               |sut                     |simzone_a                       |157.122.193.148  
|simzone_c               |5               |sut                     |simzone_c                       |18.162.176.112   
|simzone_b               |5               |sut                     |simzone_b                       |16.227.175.237   
|simzone_d               |5               |lg                      |simzone_d                       |23.123.227.142   

The VMC object simzone_c, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (VMC object key)               |value                                
|update_attempts                          |312                                  
|update_frequency                         |12                                   

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |1.55M                         
|Redis Version                                        |2.6.16                        
|Uptime (in seconds)                                  |2218156                       
|Total Connections Received                           |8101                          
|Total Commands Processed                             |2498547                       
|Number of Keys                                       |275                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.6                         
|Storage Size                                         |1585152                       
|Data Size                                            |456616                        
|Index Size                                           |408800                        
|Average Object Size                                  |506.788013319                 
|Collections                                          |13                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |0                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |12                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |8                             
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |0                             
|VMs ARRIVED                                          |0                             
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |0                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |60                            

 ---------------- VMC-WIDE COUNTERS ----------------
|22A02FDA-3999-5B5A-9EF9-0555D64855AD (simzone_c) VM RESERVATIONS|0                             
|BC2B67B7-F8FC-596D-B4B6-89781278A46F (simzone_d) VM RESERVATIONS|0                             
|31D627D0-7733-5110-A077-926AFB38B046 (simzone_b) VM RESERVATIONS|0                             
|E737205E-089F-56C4-B594-FBDFC6437474 (simzone_a) VM RESERVATIONS|0                             

