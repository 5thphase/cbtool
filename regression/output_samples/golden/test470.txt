 status: VMC 72820152-2401-5A21-B24D-A8EAFD42576E was successfully registered on SimCloud "TESTCLOUD".
 status: VMC 75153803-6653-5E26-B191-CFBC1F5A48BE was successfully registered on SimCloud "TESTCLOUD".
 status: VMC 2B308187-9A32-5BAF-AB33-1C672738FE66 was successfully registered on SimCloud "TESTCLOUD".
 status: VMC EFCC4167-8AF3-5BE8-85F2-0E6B4E5B3314 was successfully registered on SimCloud "TESTCLOUD".
 status: Attribute "collect_from_host" was set to "false". Skipping Host OS performance monitor daemon startup
All VMCs successfully attached to this experiment.
The following ATTACHED VMCs are in this experiment (Cloud TESTCLOUD) :
|name                    |host_count      |pool                    |cloud_hostname                  |cloud_ip         
|simzone_d               |5               |lg                      |simzone_d                       |218.138.158.38   
|simzone_b               |5               |sut                     |simzone_b                       |155.92.157.215   
|simzone_c               |5               |sut                     |simzone_c                       |127.96.206.118   
|simzone_a               |5               |sut                     |simzone_a                       |139.43.62.118    

The VMC object simzone_c, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (VMC object key)               |value                                
|update_attempts                          |312                                  
|update_frequency                         |12                                   

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |1.52M                         
|Redis Version                                        |2.6.16                        
|Uptime (in seconds)                                  |508294                        
|Total Connections Received                           |10579                         
|Total Commands Processed                             |819312                        
|Number of Keys                                       |275                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.9                         
|Storage Size                                         |2043904                       
|Data Size                                            |455304                        
|Index Size                                           |400624                        
|Average Object Size                                  |504.771618625                 
|Collections                                          |13                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |0                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |12                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |8                             
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |0                             
|VMs ARRIVED                                          |0                             
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |0                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |60                            

 ---------------- VMC-WIDE COUNTERS ----------------
|72820152-2401-5A21-B24D-A8EAFD42576E (simzone_a) VM RESERVATIONS|0                             
|EFCC4167-8AF3-5BE8-85F2-0E6B4E5B3314 (simzone_b) VM RESERVATIONS|0                             
|75153803-6653-5E26-B191-CFBC1F5A48BE (simzone_c) VM RESERVATIONS|0                             
|2B308187-9A32-5BAF-AB33-1C672738FE66 (simzone_d) VM RESERVATIONS|0                             

